meta-ostro/classes/ostro-image.bbclass

```diff
@@ -184,7 +184,8 @@ FEATURE_PACKAGES_ima = "packagegroup-ima-evm-utils"
 FEATURE_PACKAGES_iotivity = "packagegroup-iotivity"
 FEATURE_PACKAGES_devkit = "packagegroup-devkit \
     ${@ bb.utils.contains('IMAGE_FEATURES', 'java-jdk', 'mraa-java upm-java', '', d)} \
-    ${@ bb.utils.contains('IMAGE_FEATURES', 'python-runtime', 'python-mraa', '', d)} \
+    ${@ bb.utils.contains('IMAGE_FEATURES', 'python-runtime', 'python-mraa python-upm', '', d)} \
```

Comment:
That update was merged into ostro-os in parallel to me enabling the packages in the images - that explains why I didn't have it yet.

The commit messages in meta-intel-iot-middlware only mention the version bump, so it wasn't obvious to me that something regarding the bindings might have changed.

Has the root cause of the original problem with the Python bindings, unreliable builds, been identified and fixed?